services/spar/src/Spar/Data.hs

```diff
@@ -343,16 +349,27 @@ storeIdPConfig idp = retry x5 . batch $ do
   addPrepQuery
     byTeam
     ( idp ^. SAML.idpId,
-      idp ^. SAML.idpExtraInfo
+      idp ^. SAML.idpExtraInfo . wiTeam
     )
   where
     ins :: PrepQuery W IdPConfigRow ()
-    ins = "INSERT INTO idp (idp, issuer, request_uri, public_key, extra_public_keys, team) VALUES (?, ?, ?, ?, ?, ?)"
+    ins = "INSERT INTO idp (idp, issuer, request_uri, public_key, extra_public_keys, team, old_issuers, replaced_by) VALUES (?, ?, ?, ?, ?, ?, ?, ?)"
     byIssuer :: PrepQuery W (SAML.IdPId, SAML.Issuer) ()
     byIssuer = "INSERT INTO issuer_idp (idp, issuer) VALUES (?, ?)"
     byTeam :: PrepQuery W (SAML.IdPId, TeamId) ()
     byTeam = "INSERT INTO team_idp (idp, team) VALUES (?, ?)"
 
+markReplacedIdP ::
```

Comment:
Can we give this function a name so that it is unambigious which argument is which? or maybe a newtype so we do not mix them up? type are cheap

Reply:
79a68dbca2272ce2ff536a8ea3be1860be841635